reservation/views.py

```diff
@@ -65,21 +69,62 @@ def select_seats(request, movie, schedule):
                 details=schedule_object,
                 seat=seat_obj,
             )
+
             reservations.append(user_reservation)
```

Comment:
One thing that could be optimized here is the creation of the Reservation objects, using bulk_create.
Another one would be to use seat_pk=seat, instead of retrieving the full Seat object, when defining the Reservation objects.